web/templates/web/contributors_list.html

```
@@ -60,171 +60,263 @@ <h3 class="font-semibold text-yellow-700 dark:text-yellow-300 mb-2">
         </div>
       </div>
     </div>
-    <!-- Stats Summary Row -->
-    <div class="grid grid-cols-1 md:grid-cols-4 gap-6 mb-8">
-      <div class="bg-white dark:bg-gray-800 shadow-lg rounded-xl p-6 text-center">
-        <div class="text-4xl font-bold text-teal-300 mb-2">{{ contributors|length }}</div>
-        <div class="text-gray-600 dark:text-gray-300 font-medium">Contributors</div>
+    <!-- Year Toggle -->
+    <div class="flex justify-center mb-8">
+      <div class="inline-flex bg-gray-200 dark:bg-gray-700 rounded-xl p-1 shadow-inner">
+        <button id="toggle-2026"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)